typescript/src/index.ts

```diff
@@ -188,6 +189,24 @@ export interface TBTC {
     bitcoinClient: BitcoinClient
   ): Promise<void>
 
+  /**
+   * Prepares the proof of a redemption transaction and submits it to the
+   * Bridge on-chain contract.
+   * @param transactionHash - Hash of the transaction being proven.
+   * @param mainUtxo - Recent main UTXO of the wallet as currently known on-chain.
+   * @param walletPubKeyHash - 20-byte public key hash of the wallet
```

Comment:
I think we should use `walletPublicKey` and do the hashing inside (using `computeHash160` from `bitcoin.ts`). This way we'll have the same interface as during deposits so writing system tests may be easier as we'll need to rely just on the PK and don't bother about hashing. Also, when we document such a hex values, let's explicitly put their byte size and the note they are an un-prefixed (or 02/03 prefixed like the compressed keys) hex string.

Reply:
Done in 3491b959b2105173486d028e91926e68abfeaa6b.